BUILDING_AN_ISO.md

```diff
@@ -0,0 +1,33 @@
+# Building an ISO file
+
+To build an OpenPOWER Host OS ISO file, you'll first need to follow the
+instructions [here](README) to have a set of packages available.
+
+If you wish an installable ISO with all the built packages and all packages
+groups available during the installation, just execute:
+
+```
+python host_os.py --verbose build-iso
+```
+
+This assumes the packages are at their default location at
+`result/repository_config/latest`. The created ISO will be available in the
+result directory, and may be installed using those
+[instructions](INSTALLING_PACKAGES#installation-using-iso-file).
+
+
+## Building a reduced ISO file
+
+To build a reduced ISO file, pass the groups you wish available in the ISO to
+the `--iso-repo-additional-packages-groups` option. The unavailable groups
+and environments will not show up during the installation, so other options do
+not need to be changed.
```

Comment:
What do you mean with this last phrase? It seems this is exposing implementation details in the doc.